Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -64,8 +52,85 @@ public static class ExchangeBuffer
             var gpuBuffer = accelerator.Allocate<T>(extent);
             return new ExchangeBuffer<T>(gpuBuffer, mode);
         }
-    }
+                                        /// <summary>
```

Comment:
nit: same here; please add an empty line here to separate different method implementations. This makes it easier to read and simplifies navigation through the code using vim 🤓